src/objects/featured-row/featured-row.scss

```diff
@@ -0,0 +1,42 @@
+@use "../../compiled/tokens/scss/breakpoint";
+@use "../../compiled/tokens/scss/size";
+@use '../../mixins/fluid';
+@use '../../mixins/media-query';
+@use "../../mixins/ms";
+
+.o-featured-row {
+  display: grid;
+  @include fluid.grid-gap(
+    breakpoint.$s,
+    breakpoint.$xl,
+    size.$spacing-gap-fluid-min,
+    size.$spacing-gap-fluid-max
+  );
+
+  @media (min-width: breakpoint.$l) {
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2, max-content);
+  }
+}
+
+.o-featured-row__heading {
+  @media (min-width: breakpoint.$l) {
+    grid-column: 1 / 2;
+    grid-row: 1 / 2;
+  }
+}
```

Comment:
This is a complete nitpick, but since this class applies to an element that _contains_ a  heading (rather than directly to the heading itself), it may be better to call it something like `header` or `intro`.